src/main/java/me/juancarloscp52/entropy/events/db/AngryBeeEvent.java

```diff
@@ -17,12 +17,12 @@ public void init() {
         Entropy.getInstance().eventHandler.getActivePlayers().forEach(
                 serverPlayerEntity -> {
                     for (int i = 0; i < 3; i++) {
-                        BeeEntity bee = EntityType.BEE.spawn(serverPlayerEntity.getWorld(),serverPlayerEntity.getBlockPos().east(2), SpawnReason.SPAWN_EGG);
+                        BeeEntity bee = EntityType.BEE.spawn(serverPlayerEntity.getServerWorld(),serverPlayerEntity.getBlockPos().east(2), SpawnReason.SPAWN_EGG);
                         bee.setAngryAt(serverPlayerEntity.getUuid());
                         bee.chooseRandomAngerTime();
                     }
                 }
-        );
+                );
```

Comment:
A number of these in the PR. Worth pulling this up to the previous line if they all get poked anyway?

Reply:
Just gonna revert for now since it's a code-style thing anyway.

Reply:
Just gonna revert, as it's a code-style thing anyway.